quipucords/api/inspectresult/model.py

```diff
@@ -3,16 +3,134 @@
 These models are used in the REST definitions
 """
 
-from django.db import models
+from django.db import connection, models
+from django.db.models.expressions import RawSQL
 from django.utils.translation import gettext as _
 
 from api import messages
 from api.common.util import RawFactEncoder
 from api.source.model import Source
+from constants import DataSources
+
+RAW_QUERY_POSTGRES = """
+with raw_agg as (
+    select
+        ir.id,
+        ir.result_set_id,
+        (
+        select
+            json_object_agg(raw.name, raw.value) as raw_facts
+        from
+            api_rawfact raw
+        where
+            raw.inspect_result_id = ir.id
+    )
+from
+	api_inspectresult as ir
+)
+select
+	json_agg(raw_agg.raw_facts) as raw_fact_list
+from raw_agg
+where
+    raw_agg.result_set_id=api_resultset.id
+group by
+	raw_agg.result_set_id
+"""
+
+# RAW_QUERY_POSTGRES = """
+# with raw_agg as (
+# select
+# 	ir.id,
+# 	ir.result_set_id,
+# 	json_object_agg(raw.name, raw.value) as raw_facts
```

Comment:
Ah, perhaps this was your attempt to do what I mentioned above. I'd still like to give it a try, though. 🙂